src/middlewares/validation.js

```diff
@@ -0,0 +1,21 @@
+const validationBody = (schema) => async (req, _, next) => {
```

Comment:
why?
it`s validation middleware, which help with Joi validation somewhere.

Reply:
I delete this middleware, because with plugin `express-joi-validation` it do not need.
But this middleware we can use in every router, where we want do validation wit Joi, because I added this in this folder.

Reply:
This library has enough count of middlewares to validate different input types, so I don't think we should use this custom one

Reply:
I agree